### ISSUE TITLE
Sync with Google Calendar, Apple Cloud Calendar etc

### ISSUE THREAD

**Issue by PooSham** [WestCoastJitterbugsOrg/Personalized-Calendar#5]
While it would probably be enough to be able to export ics files as described in #4 , it would be nice to have the possibility to click a button, authorize, and get the events in your calendar.

Questions to be answered before starting: Should we create a new calendar in the account, and add the events there, or should we push them into the default calendar. Or should we let the user decide? Maybe they should be able to select whichever calendar they have.

**Comment by PooSham** [comment]
This would be a nice addition, but the questions surrounding it are hard and it doesn't seem worth the effort at the moment.
